scss/civicrm/dashboard/_dashboard.scss

```diff
@@ -0,0 +1,57 @@
+$dashboard-dashlets-margin: 10px;
+
+.crm-inactive-dashlet-fieldset {
+  legend {
+    background: none;
+  }
+
+  .crm-hover-button {
+    color: $brand-primary;
+
+    &:hover {
+      color: darken($brand-primary, $crm-darken-percentage);
+    }
+  }
+}
+
+#civicrm-dashboard {
+  .crm-dashboard-droppable {
+    margin-left: $dashboard-dashlets-margin / 2;
+    margin-right: $dashboard-dashlets-margin / 2;
+  }
+
+  .crm-dashboard-droppable:last-child {
+    margin-right: 0;
+  }
+
+  .crm-dashlet {
+    margin: $dashboard-dashlets-margin 0 0;
+  }
+
+  .crm-inactive-dashlet,
+  .crm-dashlet {
+    border-radius: $border-radius-small;
+  }
+
+  .crm-dashlet-content {
+    border-radius: 0 0 $border-radius-small $border-radius-small;
+    padding: 0;
+  }
+
+  .crm-dashlet-header {
+    background: $gray-dark;
+    border-radius: $border-radius-small $border-radius-small 0 0;
+    padding: 17px 10px;
```

Comment:
Cant we use existing values?

Reply:
This is very specific to the dashboard. There is another area that is using it:
`scss/civicrm/common/_widget-dashlet.scss`
https://github.com/civicrm/org.civicrm.shoreditch/commit/751749cd045da5a83ac126fb64ca76130e8a46e1#diff-46aa9e81be5c33d36e54dc21cd049c203726a9f35fdee9d175011dd7dba857dbR6

I suspect this is part of the dashboard and needs to be deleted since the selectors should not work, but it's not clear if this was used for widgets outside of the dashboard. The PR that introduced this change didn't explain the reason for the file and why it's under `common` instead of `dashboard`.

This will be left as something to be investigated in the future.